src/testRunner/unittests/tscWatch/programUpdates.ts

```diff
@@ -273,6 +273,46 @@ describe("unittests:: tsc-watch:: program updates", () => {
         ]
     });
 
+    verifyTscWatch({
+        scenario,
+        subScenario: "Updates diagnostics when '--allowArbitraryExtensions' changes",
+        commandLineArgs: ["-w", "-p", "/tsconfig.json"],
+        sys: () => {
+            const aTs: File = {
+                path: "/a.ts",
+                content: "import {} from './b.css'"
+            };
+            const bCssTs: File = {
+                path: "/b.d.css.ts",
+                content: "declare const style: string;"
+            };
+            const tsconfig: File = {
+                path: "/tsconfig.json",
+                content: JSON.stringify({
+                    compilerOptions: { allowArbitraryExtensions: true },
+                    files: [aTs.path],
+                })
+            };
+            return createWatchedSystem([libFile, aTs, bCssTs, tsconfig]);
+        },
+        edits: [
+            {
+                caption: "Disable  allowArbitraryExtensions",
+                edit: sys => sys.modifyFile("/tsconfig.json", JSON.stringify({
+                    compilerOptions: { allowArbitraryExtensions: false }

```

Comment:
Probably need to keep `files` in these edits

Reply:
Ugh, yes